sum-an-array/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+
+  <script>
+    var myArray = [10, 5, 3, 7, 2]
+      var i = 0;
+      var sum = 0;
```

Comment:
You don't want to have function definitions reliant on variables outside of the definition, if at all possible. This can be inside the function definition, so that the variable only exists while the function is running.